File: translation_output/mistral/codenet/python/java/p02476.java
<s> [INST] Translate the following python code to java and enclose your solution inside ```java```:
```
a,b = map(int,input().split())
print(a%b)

```
[/INST]
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        System.out.println(a % b);
    }
}
```</s>

File: translation_output/mistral/codenet/java/python/p02476.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```:
```
import java.io.BufferedInputStream;
import java.math.BigInteger;
import java.util.Scanner;

public class p02476 {
	public static void main(String []args)
	{
		Scanner in =new Scanner(new BufferedInputStream(System.in));
		while(in.hasNext())
		{
			BigInteger a = in.nextBigInteger(),b = in.nextBigInteger();
			System.out.println(a.remainder(b));
		}
		in.close();
	}
}


```
[/INST]
```python
import sys

def main():
    in = sys.stdin
    while in.has_next():
        a, b = in.next_bigint(), in.next_bigint()
        print(a % b)

if __name__ == '__main__':
    main()
```</s>
